lib/omnibus/software.rb

```diff
@@ -696,6 +750,11 @@ def overridden?
       @overrides.key?(:version) && (@overrides[:version] != @version)
     end
 
+    def build_dep
+      @build_dep
+    end
+    expose :build_dep
```

Comment:
Entirely possible @sethvargo :) I will debug through and see if it gets triggered.